### ISSUE TITLE
Question: DxDataGrid Popup as EditFormTemplate

### ISSUE THREAD

**Issue by sven5** [DevExpress/Blazor#91]
Hi folks,

what's the correct way to have a `DxPopup` as an `EditFormTemplate` ?
I already tried the following code and it seems to work. Though I'd like to have some kind of confirmation - maybe you could add a new sample for such a scenario.
```
<EditFormTemplate>
        <DxPopup Visible="true">
            <ChildContent>
                <h2>Hello worlds</h2>
            </ChildContent>
        </DxPopup>
    </EditFormTemplate>
```

**Comment by zabelin-vladimir** [comment]
Hello @sven5,

As far as I understand, you want to show DxDataGrid's edit form in our DxPopup control. We have a task to add this feature out of the box.
At the moment, you can use an approach from the [Blazor Data Grid - How to edit a row on a separate page](https://supportcenter.devexpress.com/ticket/details/t802173/blazor-data-grid-how-to-edit-a-row-on-a-separate-page) example with some changes. It's necessary to add DxPopup and show it instead of redirecting to a separate page. Please check it and let me know if it meets your requirements.

**Comment by sven5** [comment]
@zabelin-vladimir 

Ok thank you!

**Comment by zabelin-vladimir** [comment]
Hello @sven5 

I would like to note that recently [we released the v20.1.6 of our controls][0]. One of the released features is the [EditMode][1] property. You can set this property to the [DataGridEditMode.PopupEditForm][2] value, so Data Grid displays the edit form in a pop-up window. Please check the following demo illustrating different edit modes: [Data Grid - Edit Modes][3].

[0]: https://github.com/DevExpress/Blazor/releases/tag/20.1.6
[1]: https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGrid-1.EditMode
[2]: https://docs.devexpress.com/Blazor/DevExpress.Blazor.DataGridEditMode
[3]: https://demos.devexpress.com/Blazor/GridEditData#EditModes

**Comment by sven5** [comment]
Thank you @zabelin-vladimir. This is really amazing.
